app/i18n/index.js

```diff
@@ -1,36 +1,35 @@
-import i18n from 'i18next';
-import { initReactI18next } from 'react-i18next';
-import moment from 'moment';
+import i18n from "i18next";
+import { initReactI18next } from "react-i18next";
+import moment from "moment";
 
 const languages = {
-    de: { translation: require('./lang/de.json') },
```

Comment:
Please don't change the existing style. Just add your changes with the same style as the surrounding code.